### PR TITLE
Issue #13100: Create tests from LambdaBodyLength xdoc examples

### PR DESCRIPTION
**Pull request by stoyanK7**
Part of #13100

https://checkstyle.sourceforge.io/config_sizes.html#LambdaBodyLength_Examples

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+LambdaBodyLength
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.lambdabodylength;
+
+class Example1 {
+  Runnable r = () -> { // violation 'length is 11 .* (max allowed is 10).'
```
Comment (nrmancuso) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java:10]: Seems reasonable to move "good" lambda to be `r`, and place `ok` comment with explanation there.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java (hunk 1 above)
Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java:10]: We can skip `.* (max allowed is 10).` as example CI fig is above all is clear and short.

Better to not use regexp in Examples.

Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java:10]: My request above sounds like a regexp check idea for Examples. Is is doable in simple way ?

Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java]: > Seems reasonable to move "good" lambda to be r, and place ok comment with explanation there.

> We can skip .* (max allowed is 10). as example CI fig is above all is clear and short.

Done.

> My request above sounds like a regexp check idea for Examples. Is is doable in simple way?

A regexp to check for regex expressions in the `// violation` comments? We have to be very specific - for example escapes `\.`, `\+`, `\(`, `\)`, `\[`, `\]` or common expressions `.*`

Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java]: >common expressions .*

Let's target this most common mistake. We will see if we can add other also later on.

**Review by nrmancuso — APPROVED** [review]

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java (hunk 1 above)
Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java]: > Let's target this most common mistake. We will see if we can add other also later on.

https://github.com/checkstyle/checkstyle/issues/13244 is where discussion continues on this.

**Review by romani — APPROVED** [review]

**Pull request merged**